### ISSUE TITLE
Fatal error: function json_in__json_out not callable

### ISSUE THREAD

**Issue by cancilla** [IBMStreams/streamsx.topology#488]
I have the following python app that subscribes to a topic. The data being exported is in JSON format. 

```
class MyApp:
    def __init__(self, username="cancilla", password="some_pass"):
        self.username = username
        self.password = password

    def run(self):
        topo = Topology("MyApp")
        ts = topo.subscribe("the_topic", schema.CommonSchema.Json)
        ts.print()
        self.view = ts.view()

        streamsx.topology.context.submit("DISTRIBUTED", topo.graph, username=self.username, password=self.password)
```

When I submit the application, I see the following errors in the PE log.

```
[2016-08-25T09:57:37.752-0400] [9.26.94.25] [...,#splapplog,J[18],P[18],identity_2,python] Fatal error: function json_in__json_out in module streamsx.topology.runtime not callable
[2016-08-25T09:57:54.157-0400] [9.26.94.25] [...,#splapplog,J[18],P[18],identity_2,python] Fatal error: function json_in__json_out in module streamsx.topology.runtime not callable
[2016-08-25T09:57:58.381-0400] [9.26.94.25] [...,#splapplog,J[18],P[18],identity_2,python] Fatal error: function json_in__json_out in module streamsx.topology.runtime not callable
```

If I remove/comment out the line where I am creating the view (`self.view = ts.view()`) the error disappears and the application runs successfully.

**Comment by chanskw** [comment]
What happens if the subscription data is not JSON but just a string or SPL?  Would creating a view work in those cases?

**Comment by chanskw** [comment]
The other thing to note here is that the application that publishes data is a SPL application that uses the topology's Publish operator to publish data downstream for Python application to analyze.

**Comment by ddebrunner** [comment]
You can add a noop transformation before the view to work around this, something like:

```
import streamsx.topology.functions
...
ts = topo.subscribe("the_topic", schema.CommonSchema.Json)
ts = ts.map(functions.identity)
```

**Comment by ddebrunner** [comment]
This is fixed in 1.6.0 and verified still fixed in master.
